cmd/broker/binding_test.go

```diff
@@ -116,3 +119,131 @@ func TestBinding(t *testing.T) {
 	})
 
 }
+
+func TestDeprovisioningWithExistingBindings(t *testing.T) {
+	// given
+	cfg := fixConfig()
+	// Disable EDP to have all steps successfully executed
+	cfg.EDP.Disabled = true
+	suite := NewBrokerSuiteTestWithConfig(t, cfg)
+	defer suite.TearDown()
+	iid := uuid.New().String()
+	bindingID1 := uuid.New().String()
+	bindingID2 := uuid.New().String()
+
+	response := suite.CallAPI("PUT", fmt.Sprintf("oauth/v2/service_instances/%s?accepts_incomplete=true", iid),
+		`{
+					"service_id": "47c9dcbf-ff30-448e-ab36-d3bad66ba281",
+					"plan_id": "361c511f-f939-4621-b228-d0fb79a1fe15",
+					"context": {
+						"globalaccount_id": "g-account-id",
+						"subaccount_id": "sub-id",
+						"user_id": "john.smith@email.com"
+					},
+					"parameters": {
+						"name": "testing-cluster",
+						"region": "eu-central-1"
+					}
+		}`)
+	opID := suite.DecodeOperationID(response)
+	suite.processProvisioningByOperationID(opID)
+	suite.WaitForOperationState(opID, domain.Succeeded)
+
+	// when we create two bindings
+	response = suite.CallAPI("PUT", fmt.Sprintf("oauth/v2/service_instances/%s/service_bindings/%s", iid, bindingID1),
+		`{
+                "service_id": "47c9dcbf-ff30-448e-ab36-d3bad66ba281",
+                "plan_id": "361c511f-f939-4621-b228-d0fb79a1fe15"
+               }`)
+	require.Equal(t, http.StatusCreated, response.StatusCode)
+
+	response = suite.CallAPI("PUT", fmt.Sprintf("oauth/v2/service_instances/%s/service_bindings/%s", iid, bindingID2),
```

Comment:
Code duplication. Please, consider extraction of call method.

Reply:
Convention is this file is that we use CallAPI with verbose body and this was suggested as more readable here.

Reply:
There are too little tests in the file to call it a convention. Please, consider method extraction.

Reply:
See provisioning_test.go, deprovisioning_test.go, update_test.go, upgrade_test.go

Reply:
In expiration_test.go convention is slightly different.